src/CodeReview.Orchestrator/Activities/ProcessingContext.cs

```diff
@@ -9,11 +12,10 @@ public class ProcessingContext : IProcessingContext
         private readonly string _manifestFilePath;
         private readonly IContainerService _containerService;
         private readonly IPathService _pathService;
+        private readonly Dictionary<string, Volume> _volumes = new();
```

Comment:
Does it make sense use `new(StringComparer.OrdinalIgnoreCase);`? I guess this can make core developer-friendly.